app/src/main/res/values-el-rGR/strings.xml

```diff
@@ -207,7 +207,7 @@
     <string name="settings_oled_dark">Απόλυτο μαύρο φόντο για το μαύρο θέμα</string>
     <string name="settings_system_locale">Σύστημα</string>
     <string name="settings_theme_color">Χρώμα θέματος</string>
-    <string name="settings_catima_theme">Catima</string>
```

Comment:
Hey @stavpup, I'm a bit confused about this change.

In the other place you translated "Catima" as "Catima", but here you translated it as "Κάτιμα". Which is it supposed to be? Is it normal in Greece to change the writing system of brands? I know it's common in India but I didn't hear about this in Greece before.

Reply:
IMHO Catima as an app is not to be translated. The string that prefers to the theme/color is transcribed as there is no C in Greek with a K.

Reply:
That sounds strange to me, because it's the "Catima" theme and it's still the brand. But I'm not a Greek speaker so I'll just trust you :sweat_smile: